test/test_vim.py

```diff
@@ -63,8 +64,9 @@ def test_strwidth():
 @with_setup(setup=cleanup)
 def test_chdir():
     pwd = vim.eval('getcwd()')
-    vim.chdir('/')
```

Comment:
Vim/Neovim should work just fine with `/` as the root path. I just tried `:cd /` in Windows gvim, it cd'd to `C:\`.